src/connectedk8s/azext_connectedk8s/custom.py

```diff
@@ -301,6 +301,10 @@ def create_connectedk8s(
     ):
         lowbandwidth = True
 
+    azure_local_disconnected = False
+    if os.getenv("AZURE_LOCAL_DISCONNECTED") == "true":
+        azure_local_disconnected = True
```

Comment:
```suggestion
    azure_local_disconnected = ( os.getenv("AZURE_LOCAL_DISCONNECTED") == "true" )
```

Reply:
> os.getenv("AZURE_LOCAL_DISCONNECTED") == "true"

Do we need to consider ignoring capitalization when comparing

Reply:
On Windows, the environment variables are case insensitive and on Linux the environment variables are case sensitive. So, we should adjust to the most sensitive usage. `os.getenv()` is by default already case sensitive.